server/odc-service/src/main/java/com/oceanbase/odc/service/task/executor/Task.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) 2023 OceanBase.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.oceanbase.odc.service.task.executor;
+
+import com.oceanbase.odc.core.flow.model.FlowTaskResult;
+import com.oceanbase.odc.core.shared.constant.TaskStatus;
+import com.oceanbase.odc.service.task.caller.JobContext;
+
+/**
+ * Task interface. Each task should implement this interface
+ * 
+ * @author gaoda.xy
+ * @date 2023/11/22 15:46
+ */
+public interface Task {
+
+    /**
+     * Start current task. This method will be called by {@link TaskExecutor} for fire a task
+     */
+    void start();
```

Comment:
what is the behaivior of `start()` method, block until complete or return straightly ?

Reply:
The `start()` method itself in `Task` implementation is blocked until complete. But we can execute this method in a `Executor` (such as `executor.submit(task::start)`) and get empty return straightly.